instance/tests/management/test_instance_redeploy.py

```diff
@@ -247,12 +244,8 @@ def _spawn_appserver_success(
             # Verify the logs
             captured_logs.check(*expected_logs)
 
-        # Ensure that the load balancer was reconfigured only once per batch of successful appservers
-        self.assertEqual(mock_reconfigure.call_count, 2)
```

Comment:
Is there no way to test the LB re-configuration anymore here?

Reply:
Since we're now delegating the LB re-configuration logic to `make_active`, which is itself now being called only in `spawn_appserver`, which we're mocking above, we can't really test this, no. But tests for `spawn_appserver` elsewhere should give us confidence that `make_active` is called, and tests for `make_active` should give us confidence that the LB is reconfigured.